### DISCUSSION TITLE
libritts_r samples download

### DISCUSSION THREAD

**Discussion by omega3** [rhasspy/piper#405] [Q&A] [answered]
For US libritts_r (medium) there are 904 speakers. I use [this](https://rhasspy.github.io/piper-samples/) dedicated site to listen to them but there are to many. It would be more convenient for me to download these samples and listen in a player.  There is an [address](http://www.openslr.org/141/) given but I don't know what specific file to download. Please, can you point the one that is used on sample site?

**Comment by synesthesiam** [comment, marked as answer]
Sure, those samples are available here: https://github.com/rhasspy/piper-samples/tree/master/samples/en/en_US/libritts_r/medium
